dags/socorro_import.py

```diff
@@ -143,19 +142,20 @@
     ]
 
     # We remove the current date partition for idempotency.
-    remove_bq_table_partition = BigQueryDeleteTableOperator(
-        task_id="remove_bq_table_partition",
+    table_name = "{}:{}.{}${{{{ds_nodash}}}}".format(
+            "{{ var.value.gcp_shared_prod_project }}", bq_dataset, bq_table_name)
+
+    remove_bq_table_partition = GKEPodOperator(
+        task_id="remove_socorro_crash_bq_table_partition",
         gcp_conn_id=bq_gcp_conn_id,
-        deletion_dataset_table="{}.{}.{}${{{{ds_nodash}}}}".format(
-            "{{ var.value.gcp_shared_prod_project }}", bq_dataset, bq_table_name
-        ),
-        ignore_if_missing=True,
+        name="remove_socorro_crash_bq_table_partition",
+        image="gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine",
```

Comment:
I think we should avoid relying on an external image, bqetl has this installed:
```suggestion
        image="gcr.io/moz-fx-data-airflow-prod-88e0/bigquery-etl:latest",
```
Or mirror this image onto our infra.

Reply:
testing now